nifi-system-tests/nifi-system-test-suite/src/test/resources/conf/clustered/node2/logback.xml

```diff
@@ -84,13 +84,13 @@
             <maxHistory>5</maxHistory>
         </rollingPolicy>
         <encoder class="ch.qos.logback.classic.encoder.PatternLayoutEncoder">
-            <pattern>%date %level [%thread] %logger{40} %msg%n</pattern>
+            <pattern>%date %level [%thread] %logger %msg%n</pattern>
         </encoder>
     </appender>
 
     <appender name="CONSOLE" class="ch.qos.logback.core.ConsoleAppender">
         <encoder class="ch.qos.logback.classic.encoder.PatternLayoutEncoder">
-            <pattern>%date %level [%thread] %logger{40} %msg%n</pattern>
+            <pattern>%date %level [%thread] %logger %msg%n</pattern>
```

Comment:
The `{40}` makes it harder to grep log files for log messages coming from a given package.